src/main/java/exceptions/InvalidCommandException.java

```diff
@@ -0,0 +1,5 @@
+package exceptions;
+
+public class InvalidCommandException extends Exception {
+
```

Comment:
you can consider exploiting the custom exception to do something specific to Duke; perhaps a customized error message?